test/plan-marshall/workflow-integration-git/test_worktree_remove_cwd_geometry.py

```
@@ -0,0 +1,697 @@
+)
+
+
+def _init_main_repo(main: Path) -> None:
+    """Seed a real main checkout whose ``.gitignore`` covers ``.plan/``.
+
+    The ignore rule matters: worktree-resident plan state is untracked, and
+    ``git worktree remove`` without ``--force`` refuses a worktree carrying untracked
+    files. Ignoring ``.plan/`` keeps the plan state from blocking the non-force removal,
+    so the only thing that can refuse is one of the two preconditions under test.
+    """
+    subprocess.run(['git', 'init', '-q', '-b', 'main', str(main)], check=True)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)